helmfile/02-networkpolicies.yaml

```
@@ -34,7 +34,7 @@ releases:
   chart: ./charts/networkpolicy-common
```

Do the same thing for the chart?
```suggestion
  chart: ./charts/networkpolicy/common
```